src/frameworks/next/index.ts

```diff
@@ -222,6 +285,29 @@ export async function ɵcodegenPublicDirectory(sourceDir: string, destDir: strin
           continue;
         }
 
+        if (rewritesNotSupportedByFirebase) {
```

Comment:
If we destructure / guard, then we can simplify this (and the others) as

```ts
if (rewritesNotSupportedByFirebase.find(rewrite => new RegExp(rewrite.regex).test(path))) {
  continue;
}```

Reply:
As an optimization you would pre init all the regex before the `for`, `e.g, const rewriteRegexsNotSupportedByFirebase = rewritesNotSupportedByFirebase.map(rewrite => new RegExp(rewrite.regex))`

Reply:
@jamesdaniels I changed that to destructure with defaults and also instantiated the regexes before the loop as suggested.

Then instead of
```js
if (rewritesNotSupportedByFirebase.find(rewrite => new RegExp(rewrite.regex).test(path))) {
  continue;
}
```

I assigned the loop result to a variable because I think it's easier/quicker to understand as the variable name replaces a comment that may become outdated if this changes at some point. Also as the name is quite long, IMO a variable name can help. Let me know your thoughts